.flake8

```diff
@@ -1,3 +1,4 @@
 [flake8]
 max-line-length=120
 exclude = .git,__pycache__,dist,.venv/*,node_modules/*,utils/*,tests/.venv/*
+extend-ignore = F401,F403,F405
```

Comment:
What do these codes correspond to and why do we ignore them?

Reply:
have removed them as was not required